ci-operator/config/openshift/cloud-provider-gcp/openshift-cloud-provider-gcp-main.yaml

```diff
@@ -133,11 +133,13 @@ tests:
       PROJECT_NAME: cloud-provider-gcp
     workflow: openshift-ci-security
 - as: verify-deps
-  steps:
-    env:
-      CHECK_MOD_LIST: "false"
-    test:
-    - ref: go-verify-deps
```

Comment:
What does `go-verify-deps` do differently?

Reply:
https://github.com/openshift/cloud-provider-gcp/blob/main/openshift-hack/update-vendor.sh#L29-L34 

This approach is lifted from upstream, but with bazel removed.